fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -28,4 +28,5 @@
       # queue limit is hit - 'block' will halt further reads and keep retrying to flush the
       # buffer to the remote - default is 'exception' because in_tail handles that case
       buffer_queue_full_action "#{ENV['BUFFER_QUEUE_FULL_ACTION'] || 'exception'}"
+      ssl_verify "#{ENV['ES_COPY_SSL_VERIFY'] || 'true'}"
```

Comment:
Why do we need to expose out to the product a feature used for testing?  Doesn't this mean that once released, I can modify my DC to turn off ssl verification via the ES_COPY_SSL_VERIFY environment variable?

Since this feature is being removed, why do this on master, and add a new parameter?  Can we test this a different way?